recipes/pyexecjs/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyexecjs" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyExecJS-{{ version }}.tar.gz
+  sha256: 34cc1d070976918183ff7bdc0ad71f8157a891c92708c00c5fbbff7a769f505c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - six >=1.10.0
+
+test:
+  imports:
+    - execjs
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/doloopwhile/PyExecJS
+  summary: Run JavaScript code from Python
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - UlionTse
+    # - doloopwhile
```

Comment:
Don't add other people, not even as a comment, please.
```suggestion
```

Reply:
OK